src/lib/HST/Util/Selectors.hs

```diff
@@ -96,94 +95,108 @@ getPatVarName (S.PList _ _) = reportFatal
   $ "Expected variable or wildcard pattern, got list pattern."
 
 -------------------------------------------------------------------------------
--- Find identifiers in a programs                                                             --
+-- Find Identifiers in Modules                                               --
 -------------------------------------------------------------------------------
+-- Class for all types that contain identifiers.
+class HasIdentifiers a where
+  findIdentifiers :: a -> Set String
+
+-- Collects all identifiers in a list of types containing identifiers
+instance HasIdentifiers a => HasIdentifiers [a] where
+  findIdentifiers = Set.unions . map findIdentifiers
+
+-- Collects all identifiers in a `Maybe` value if a value is present.
```

Comment:
Note: The comment for the `Maybe` instance implied that the instance was not defined for `Nothing`.
```suggestion
-- | Class for all types that contain identifiers.
class HasIdentifiers a where
  findIdentifiers :: a -> Set String

-- | Collects all identifiers in a list of types containing identifiers.
instance HasIdentifiers a => HasIdentifiers [a] where
  findIdentifiers = Set.unions . map findIdentifiers

-- | Collects all identifiers in a `Maybe` value.
```